src/main/java/br/com/zup/edu/biblioteca/validators/ValidacaoParaEmprestimoPorTipoDeUsuarioPadrao.java

```diff
@@ -0,0 +1,91 @@
+
+import javax.persistence.EntityManager;
+import javax.persistence.TypedQuery;
+import java.util.Objects;
+
+import static br.com.zup.edu.biblioteca.model.TipoCirculacao.LIVRE;
+import static br.com.zup.edu.biblioteca.model.TipoUsuario.PADRAO;
+import static javax.persistence.LockModeType.READ;
+//10
+@Component
+public  class ValidacaoParaEmprestimoPorTipoDeUsuarioPadrao implements ValidacaoEmprestimo{
+
+    //1
+    private final ExecutorTransacional execTransacional;
+
+    public ValidacaoParaEmprestimoPorTipoDeUsuarioPadrao(ExecutorTransacional execTransacional) {
+        this.execTransacional = execTransacional;
+    }
+
+
+    @Override
+    public Errors handler(Errors errors, CadastroEmprestimoDeExemplarRequest request) {
+        final EntityManager manager = execTransacional.getManager();
+        Usuario possivelResponsavelPeloEmprestimo = execTransacional.executor(()->manager.find(Usuario.class, request.getIdUsuario()));
+        //1
+        if(Objects.isNull(possivelResponsavelPeloEmprestimo)){
+            errors.rejectValue("idUsuario",null,"Usuario nao cadastrado");
+            return errors;
+        }
+
+        //1
+        Livro livroDesejado=execTransacional.executor(()-> manager.find(Livro.class,request.getIdLivro()));
+
+        if(Objects.isNull(livroDesejado)){
+            errors.rejectValue("idLivro",null,"Livro nao cadastrado");
+            return errors;
+        }
+
+        String porQuantosExemplaresEsteUsuarioEResponsavel = "SELECT COUNT(e) FROM EmprestimoDeExemplar e WHERE e.usuario=:responsavel";
+        TypedQuery<Long> query = manager.createQuery(porQuantosExemplaresEsteUsuarioEResponsavel, Long.class);
+        query.setParameter("responsavel", possivelResponsavelPeloEmprestimo);
+        Integer quantidadeDeLivrosEmprestados = execTransacional.executor(()->query.getSingleResult().intValue());
+
+        //1
+        if (possivelResponsavelPeloEmprestimo.getTipoUsuario().equals(PADRAO)) {
+            //1
+            if (Objects.isNull(request.getTempoDeEmprestimoEmDias())) {
+                errors.rejectValue("tempoDeEmprestimoEmDias",null,"O Emprestimo de exemplares para usuarios do tipo padrao deve conter o tempo que pretende ficar com o livro");
+                return errors;
+            }
+            //1
+            else if (request.getTempoDeEmprestimoEmDias() > 60) {
+                errors.rejectValue("tempoDeEmprestimoEmDias",null,"O Emprestimo de exemplares deve ter no maximo tempo de 60 dias");
+                return errors;
+            }
+            //1
+            else if (quantidadeDeLivrosEmprestados > 4) {
+                errors.rejectValue("idUsuario",null,"Usuarios do tipo padrão podem ter no maximo 5 livros emprestados");
+                return errors;
+            }
+
+
+            String busqueAquantidadeDeExemplaresLivres = "SELECT COUNT(e) FROM Exemplar e WHERE e.tipoCirculacao=:livre AND e.emprestado=:falso and e.livro=:livro";
```

Comment:
Isso aqui já é uma outra validação. Se a gente for olhar o SOLID, a gente tá ferindo um pouco o SRP, concorda?